benchmarks/kernels/benchmark_mixtral_moe_rocm.py

```diff
@@ -393,8 +244,8 @@ def run_timing(
             config,
             compute_type=(tl.bfloat16 if hidden_states.dtype == torch.bfloat16
                           else tl.float16),
-            use_fp8=False,
-        )
+            use_fp8_w8a8=False,
+            use_int8_w8a16=False)
```

Comment:
It is probably beneficial to become explicit in the naming here.

From the current notation it is not entirely clear what "a16" is referring to. Is it "int16", "fp16" or "bf16" ?

All of the above-mentioned ways to represent activations could (at least theoretically) exist. Thus, making a more detailed name may save us some effort going forward.

Reply:
The name is kept in accordance with the[ upstream api call of the fused_moe kernel.](https://github.com/vllm-project/vllm/blob/d9cd78eb718c233ebc5b84377fc2226af7ef0fa2/vllm/model_executor/layers/fused_moe/fused_moe.py#L58)

Some digging on the int8_w8a16 format: this refers to "weight-only" quantization and the activations can be preserved either in fp16 or bf16 ([link](https://nvidia.github.io/TensorRT-LLM/reference/precision.html#int4-and-int8-weight-only-w4a16-and-w8a16))

Reply:
OK! Thanks for clarification. Makes total sense to me.